utils/products.js

```diff
@@ -1,152 +1,372 @@
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
+  },
+  {
+    productName: "Policy Hub",
+    productDescription:
+      "The Policy Hub enables data providers, consumers and app providers to access a single-point-of-truth for Catena-X policies, attributes and templates for policy rules.",
+    githubRepo: ["https://github.com/eclipse-tractusx/policy-hub"],
+    committers: [
+      "https://github.com/evegufy",
+      "https://github.com/Phil91",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Tractus-X Policy Hub Team",
+    hasBoard: false,
+    showVersion: true,
   },
-
   {
     productName: "Portal & Marketplaces",
-    productDescription: "The Portal facilitates the operations for dataspace members (companies), it includes functionalities such as registration, technical onboarding and marketplaces.",
+    productDescription:
+      "The Portal facilitates the operations for dataspace members (companies), it includes functionalities such as registration, technical onboarding and marketplaces.",
     githubRepo: [
       "https://github.com/eclipse-tractusx/portal",
       "https://github.com/eclipse-tractusx/portal-backend",
       "https://github.com/eclipse-tractusx/portal-frontend",
       "https://github.com/eclipse-tractusx/portal-frontend-registration",
       "https://github.com/eclipse-tractusx/portal-assets",
-      "https://github.com/eclipse-tractusx/portal-iam"
+      "https://github.com/eclipse-tractusx/portal-iam",
+      "https://github.com/eclipse-tractusx/portal-shared-components",
     ],
     committers: [
       "https://github.com/evegufy",
       "https://github.com/ntruchsess",
       "https://github.com/oyo",
-      "https://github.com/Phil91"
+      "https://github.com/Phil91",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Portal and Marketplaces Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Predictive Unit Real-Time Information Service",
+    productDescription:
+      "Predictive Unit Real-Time Information Service (PURIS) for Short Term Demand and Capacity Management",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/puris",
+    ],
+    committers: [
+      "https://github.com/tom-rm-meyer-ISST",
+      "https://github.com/mhellmeier",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Portal and Marketplaces Team",
+    mailTo: "tractusx-dev@eclipse.org?subject=Request puris Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "quality-dashboard",
+    productDescription: "TBD",
+    githubRepo: ["https://github.com/eclipse-tractusx/quality-dashboard"],
+    committers: ["https://github.com/tomaszbarwicki"],
+    mailTo: "tractusx-dev@eclipse.org?subject=Request quality-dashboard Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
   },
-
   {
     productName: "Semantic Layer & Digital Twin",
```

Comment:
Where does this project come from? It has been a team and this is a summary of all products belonging to the team. 

What's this list's target? If it's about listing products and repositories I would expect them to be separated because e.g. an adopter does not need to use all of these services to have one application runnint (e.g. frontend, backend, documentation, etc. for portal). Then only the leading repository of the  `.tractusx` needs to be listed.

Reply:
This has been split. Still not sure if it's perfect, but good to go.